packages/enzyme-example-mocha/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "enzyme-example-mocha",
+  "version": "0.1.0",
+  "description": "Example project with React + Enzyme + Mocha",
+  "scripts": {
+    "test": "mocha --require test/.setup.js --compilers js:babel-core/register src/*.spec.js",
+    "test-watch": "npm test -- --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/airbnb/enzyme.git"
+  },
+  "author": "Leland Richardson <leland.richardson@airbnb.com>",
+  "license": "MIT",
+ 
+  "homepage": "https://github.com/airbnb/enzyme.git",
+  "devDependencies": {
+    "babel": "^6.23.0",
+    "babel-core": "^6.26.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "chai": "^3.5.0",
+    "enzyme": "^3.0.0",
+    "enzyme-adapter-react-16": "^1.0.0",
+    "jsdom": "^11.3.0",
+    "mocha": "^3.5.3",
+    "react-test-renderer": "^16.0.0"
```

Comment:
this might actually not be required anymore, since #1234

Reply:
I removed react-test-renderer